docs/source/notebooks/wps_climdex_su_demo.ipynb

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
<p>Wrong bird!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/quail/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
That probably explains why `make test-notebooks-prod` wasn't working :sweat_smile:

Reply:
> That probably explains why `make test-notebooks-prod` wasn't working

I believe the command would not work until we release a new version for `quail`.

Reply:
> That probably explains why `make test-notebooks-prod` wasn't working

I believe the command would not work until we release a new version for `quail`.

Reply:
That was just related to some debugging Nik was helping me with because I thought there was an issue with the docker container, but it was actually the mimetype